### PR TITLE
redpanda: 22.3.11 -> 22.3.13

### PR DESCRIPTION
**Pull request by avakhrenev**
Build `rpk` redpanda client from source.

Add `redpanda` server, which is also built from source. It works only on `linux` (because of [`dpdk`](https://github.com/spdk/spdk/issues/2#issuecomment-276127149) dependency), but on MacOS one can run redpanda via `rpk container start `.

###### Description of changes

[`redpanda 22.3.13` Release Notes](https://github.com/redpanda-data/redpanda/releases/tag/v22.3.13)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -1417,6 +1417,12 @@
     githubId = 12958979;
     name = "Mika Naylor";
   };
+  avakhrenev = {
```
Comment (happysalada) [inline maintainers/maintainer-list.nix:1420]: there is probably no need here, but usually adding yourself as a maintainer is done in a separate commit, just in case a commit need to be reverted.

**Comment by happysalada** [comment]
This is looking really good, thanks! I'll wait for the eval to finish just to make sure everything is in order.

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (avakhrenev) [reply maintainers/maintainer-list.nix:1420]: Should I remove myself and make a separate PR?

**Comment by avakhrenev** [comment]
@happysalada thank you!

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (happysalada) [reply maintainers/maintainer-list.nix:1420]: it's just a detail, I'm mentioning it here to make sure the information is not lost, but honestly, It's a detail.
